packages/react/src/Breadcrumbs/Breadcrumbs.module.css

```diff
@@ -9,6 +10,100 @@
   margin-bottom: 0;
 }
 
+[data-overflow='menu'] .BreadcrumbsList,
+[data-overflow='menu-with-root'] .BreadcrumbsList {
+  white-space: nowrap;
+  display: flex;
+  flex-direction: row;
+}
+
+.BreadcrumbsItem {
+  display: inline-grid;
+  grid-auto-flow: column;
+  align-items: center;
+  flex: 0 99999 auto;
+  min-width: auto;
+  font-size: var(--text-body-size-medium);
+  white-space: nowrap;
```

Comment:
This `white-space: nowrap` is bleeding into the items in the overflow menu from this declaration, but also from another spot in here (I think). Something nested.. but anyway, we don't want this on the items inside the menu. They should be able to wrap line.